### DISCUSSION TITLE
Introductions

### DISCUSSION THREAD

**Discussion by webcrumbs-community** [webcrumbs-community/webcrumbs#16] [General]
A thread where community members can introduce themselves.

**Comment by Ashok-Kumar-dharanikota** [comment]
Hi @webcrumbs-community ,
First of all, I must say this project is a wonderful idea. creating a no-code platform to create a react web application.

I went through all the discussions and codebase too and I like to contribute my skills like React, JavaScript, etc., and am willing to learn the other tech stack mentioned in discussions. I would like to work on some good first issues and also maintain constant discussions with you to learn more about the project. 

I would like to know the high and low-priority contributions of this project. Those who are well-known about the stack will work on high-priority tasks and meanwhile, I will work on small ones and also learn what they doing.  What do you say?

**Reply by webcrumbs-community** [reply]
Hello @Ashok-Kumar-dharanikota. Welcome aboard!

Please give the project a 🌟 so we can continue growing our community.

In this first month, we're gathering interested people to join forces. Feel free to invite friends or post about the project. Let's see if we can get to a 100 to confirm the wider value of our project.

A quick outline of the main topics in the project:
- frontend of the admin panel (nextjs)
- mechanics of third party plugins (webpack)
- drag and drop interface to design websites (?)

It's to be detailed. I think as we advance in these major topics the smaller issues will appear.

We still need to research what are the options for a drag and drop interface. See what others have been doing in this regard, check the underlying technology, learn what's available as open source.

Cheers, friend!

**Comment by AJBrownson** [comment]
Hello guys
I like the way this project seeks to fill a gap in an industry most people didn't know existed

I would really love to be a part of this innovation

**Reply by webcrumbs-community** [reply]
Welcome, AJ! Great having you here. Feel free to comment on the discussions, propose issues and volunteer to solve them!
Also bring your friends on. Let's build the next big thing together 💪

**Comment by zuxcode** [comment]
Hello, everyone!

I'm thrilled to join this vibrant community of developers and enthusiasts.

**Reply by webcrumbs-community** [reply]
Hello, there, @Codeauthor1 ! 
You're very welcome, my friend.

First make sure to 🌟 star our repo and invite your friends, so we can continue growing our community!

Check our [roadmap](https://github.com/webcrumbs-community/webcrumbs/wiki/Roadmap), our [discussions](https://github.com/webcrumbs-community/webcrumbs/discussions), and our [issues](https://github.com/webcrumbs-community/webcrumbs/issues). We add new issues almost every day.

Let us know if there's a special area you'd like to contribute.

💪💪💪

<a href="https://github.com/webcrumbs-community/webcrumbs">
  <img
    src="https://github.com/webcrumbs-community/webcrumbs/blob/main/src/img/meme.png?raw=true"
    alt="WebCrumbs Contributor"
    width="400px"
  />
</a>

**Reply by webcrumbs-community** [reply]
(Btw, @Codeauthor1 i loved your badge!)

**Reply by zuxcode** [reply]
Thank you

**Comment by aditya4196** [comment]
@webcrumbs-community 
I have a suggestion regarding the feature to drag drop interface, we could design a widget components panel where you can choose whichever component you want to accommodate in your UI and then use some dynamic grid library which will help you drag and drop the components to whichever position your want on the UI.

If this idea seems good, we would like to work on it (we are a group of programmers who would work on implementing it)

**Comment by rovoskarl** [comment]
Hello everyone
I'm happy to join this community.
I am an experienced front-end engineer.

**Reply by webcrumbs-community** [reply]
Welcome, @rovoskarl! Glad you're here. Your support will be very welcome.

**Comment by WassCodeur** [comment]
Hello everyone! 
happy to be here with you. I'd like to contribute to this great project.

**Reply by webcrumbs-community** [reply]
You're very welcome, @WassCodeur! Please give the project a 🌟 so we can continue growing our community. Feel free to comment on the discussions, propose issues and volunteer to solve them! Also bring your friends on. Let's build the next big thing together 💪

**Comment by jsimples** [comment]
Hi, everyone! I'm more than excited too about being a part of the history of WebCrumbs. I see a great future ahead!

@Ashok-Kumar-dharanikota @AJBrownson and @Codeauthor1, I'm working on the frontend of webcrumbs.org and would love to put your comments there, if you agree. It would look something like [this](https://anysite-2dd7be.webflow.io/). It's still not ready, but I'd like to ask your permission for mentioning you! Ask me if you'd like any change or removal. I'd be glad to do it!!

**Reply by WassCodeur** [reply]
the site is very beautiful. simple but impactful.

**Reply by aditya4196** [reply]
Site looks clean!!
I would also like to contribute to the frontend site!!
Please do let me know if I can work on a module!!

**Reply by Ashok-Kumar-dharanikota** [reply]
wonderful @justicasimples 🤩.

At starting. 
where the mobile and other stuff floating. What it would be like if we applied border-radius to it? 🤔

**Reply by AJBrownson** [reply]
@justicasimples nice
Love the simplicity of the site

**Reply by AJBrownson** [reply]
@justicasimples I have a suggestion
You could put those countries flags in a carousel so that a user can see them all

**Reply by jsimples** [reply]
Great feedback, everyone!! I'm opening a new discussion for this topic only.

**Comment by mohit-kota** [comment]
Hello Team 👋

**Reply by Ashok-Kumar-dharanikota** [reply]
Hi @mohit-kota 

Welcome to the Webcrumbs Community 🤝

**Reply by mohit-kota** [reply]
Hi @Ashok-Kumar-dharanikota

**Reply by webcrumbs-community** [reply]
Hi, @mohit-kota! Welcome aboard.

Check our [roadmap](https://github.com/webcrumbs-community/webcrumbs/wiki/Roadmap), our [discussions](https://github.com/webcrumbs-community/webcrumbs/discussions), and our [issues](https://github.com/webcrumbs-community/webcrumbs/issues). We add new issues almost every day.

Let us know if there's a special area you'd like to contribute.

🚀🚀🚀

**Comment by ShatilKhan** [comment]
Hi guys
Exploring & learning open source
I find webcrumbs very interesting 
Hope to contribute frontend plugins
Please let me know if there's any issues where I can help

**Comment by webcrumbs-community** [comment]
Welcome, ShatilKhan! 
* Comment on #115 so you can receive updates on frontend issues! We should have new issues soon
* You're welcome to join our discord if you'd like [WebCrumbs Discord](https://discord.com/invite/ymVgdPe6rK)

Let's build together!

**Comment by lyubomirvaklinov** [comment]
Hello, @webcrumbs-community team! Amazing idea, hope I can be of some sort of help to the community.

**Reply by webcrumbs-community** [reply]
Hey, @lyubomirvaklinov! Welcome, my friend. Let's us know what you like doing in webdev.

**Comment by messaismael** [comment]
Hi @webcrumbs-community,

It's a pleasure for me to be a part of this great community, I like the idea behind the project it's awesome 🚀 🚀  and I would like to give my part contribution to this project.

**Reply by webcrumbs-community** [reply]
Hi, @messaismael! Thanks for joining this mission. You're very welcome! What do you like doing the most in webdev?

**Reply by messaismael** [reply]
Thank you @webcrumbs-community! 
I'm more comfortable on the Frontend side and I can also handle some backend stuff.
